Dockerfile.d/etc_containerd_config.toml

```diff
@@ -5,3 +5,8 @@ version = 2
   [proxy_plugins.stargz]
     type = "snapshot"
     address = "/run/containerd-stargz-grpc/containerd-stargz-grpc.sock"
+
+# Enable soci snapshotter
+  [proxy_plugins.soci]
```

Comment:
One last comment. Unfortunately `etc_containerd_config.toml` is copied in BASE stage of Dockerfile so I believe it would change the config in the final released nerdctl-full. https://github.com/containerd/nerdctl/blob/8cef5e9166c76303f444bd36001e4b6eadaa19e3/Dockerfile#L254

We can't directly append SOCI config in the same file.